src/test/java/stringcalculator/StringParser.java

```diff
@@ -0,0 +1,29 @@
+package stringcalculator;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringParser {
+    private static final String DELIMITER = ",|:";
+    private static final Pattern CUSTOM_PATTERN = Pattern.compile("//(.)\n(.*)");
+    private static final int SEPARATOR_GROUP_NUMBER = 1;
+    private static final int EXPRESSION_GROUP_NUMBER = 2;
+
+    public static int[] parser(String s) {
+        Matcher matcher = CUSTOM_PATTERN.matcher(s);
+        if (matcher.find()) {
+            return split(matcher.group(EXPRESSION_GROUP_NUMBER), String.join("|", DELIMITER, matcher.group(SEPARATOR_GROUP_NUMBER)));
+        }
+
+        return Arrays.stream(s.split(DELIMITER)).
+                mapToInt(Integer::parseInt).
+                toArray();
```

Comment:
일반적으론 줄바꿈 후에 점(.)을 찍습니다...ㅎㅎ
```suggestion
        return Arrays.stream(s.split(DELIMITER))
                .mapToInt(Integer::parseInt)
                .toArray();
```

Reply:
수정했습니다.! 👀